### DISCUSSION TITLE
Labs for HERO13 1.20 is coming

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#1130] [Announcements]
The Labs FW for 1.20 is still in testing, but will be out within a week I expect.  The 1.20 mainline firmware is primarily a bug fix addition. Labs extensions are mainly incremental.  Here is the expected change list

- Added *DRFT=1 for automatic clock fix.
- Added *DRFS=-11.3 for a fixed clock fix.
- Added mPp1 or p.5 command for interval photo settings
- Fix for the TONE extension (was working)
- Fix for wake events under power (was a missed condition)
- Fix for ISO and SHUT LCD overlay position 
- Fix for multiple day wake events !259200N <- sleep for 72 hours.
- Fix for the !1R required in motion detection, now !R works again 

DRFT is a automatic self calibrating fix for time and date on your camera. If you sync time on your camera twice, say one week apart, it will observe the drift and compute the required correction. You should sync with the same time source using https://gopro.github.io/labs/control/precisiontime/ 

DRFS allows you to manually compensate for any observed drift. Most H12s and H13s are around 11s fast per day. I have just completed a 6 month long timelapse with HERO12, and my clock had drift 30 minutes. To fix this I can now use *DRFS=-11.1 (or *DRFT=1 for automatic calibration).  

On HERO9/10/11/13, using *SYNC=1 will use the GPS to fix the clock, but the above works without GPS enabled.

All of this is coming to HERO12 soon too.

**Comment by kouyoumin** [comment]
Interesting, but what if syncing period is short, say, seconds?
Camera may sync several times when scanning precision time QR code.
Clock jitter can be more significant when calculating 'drift / time between sync'.

**Comment by dnewman-gpsw** [comment]
there is a test for a minimum 8 hours between sync be drifts are computed, 24 hours is better.

**Comment by dnewman-gpsw** [comment]
The release is now available.
